sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/CheckpointStore.cs

```diff
@@ -99,15 +107,15 @@ public abstract class CheckpointStore
         /// <param name="consumerGroup">The name of the consumer group the checkpoint is associated with.</param>
         /// <param name="partitionId">The identifier of the partition the checkpoint is for.</param>
         /// <param name="clientIdentifier">The unique identifier of the client that authored this checkpoint.</param>
-        /// <param name="checkpointStartingPosition">The starting position to associate with the checkpoint, indicating that a processor should begin reading from the next event in the stream.</param>
+        /// <param name="startingPosition">The starting position to associate with the checkpoint, indicating that a processor should begin reading from the next event in the stream.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken" /> instance to signal a request to cancel the operation.</param>
         ///
         public virtual Task UpdateCheckpointAsync(string fullyQualifiedNamespace,
                                                   string eventHubName,
                                                   string consumerGroup,
                                                   string partitionId,
                                                   string clientIdentifier,
-                                                  CheckpointPosition checkpointStartingPosition,
+                                                  CheckpointPosition startingPosition,
```

Comment:
Since this one is new, we should make the cancellation token optional here for sure, as we won't break anyone.

Reply:
Should this be optional in all the new overloads then?

Reply:
Yeah, it was a copy/paste mistake that the original implementation wasn't.